docker/Dockerfile

```diff
@@ -30,9 +30,19 @@ RUN /tmp/set_up_stackwalker.sh && \
 COPY --chown=app:app ./webapp/package*.json /webapp-frontend-deps/
 RUN cd /webapp-frontend-deps/ && npm install
 
-COPY --chown=app:app requirements.txt /app/
+# NOTE(relud): these dependencies are installed separately, relocated, and patched to reference the
+# new location (aka shaded), so that they can be installed at the same time as newer versions
+COPY --chown=app:app legacy-es-requirements.txt /app/
 RUN pip install -U 'pip==24.0' && \
-    pip install --no-cache-dir --no-deps -r requirements.txt && \
+    pip install --no-cache-dir --no-deps -r legacy-es-requirements.txt && \
+    cd /usr/local/lib/python3.11/site-packages/ && \
+    mv elasticsearch elasticsearch_1_9_0 && \
+    mv elasticsearch_dsl elasticsearch_dsl_0_0_11 && \
+    cd elasticsearch_dsl_0_0_11/ && \
+    sed 's/^from elasticsearch/from elasticsearch_1_9_0/' -i connections.py search.py serializer.py
```

Comment:
If it were me, this is complex enough to warrant sticking in a shell script. That makes it easier to lint, test, and verify. We do this with setting up the stackwalker and ubuntu.

This is temporary, though, so if you want to leave it, that's fine.